docs/maintainers/README.md

```diff
@@ -98,15 +98,15 @@ In these cases:
 
 ## Becoming a Repository Maintainer
 
-Repository Maintainers currently consist entirely of Microsoft employees.
+Repository Maintainers currently consist mostly of Microsoft employees.
 It is expected that over time, regular trusted contributors to the PowerShell repository will be made Repository Maintainers.
-Eligibility is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways to the project will be recognized accordingly.
+Eligibility is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways consistently to the project will be recognized accordingly.
 
-At any point in time, a Repository Maintainers can nominate a strong community member to become a Repository Maintainer.
-Nominations should be submitted in the form of [RFCs][RFC-repo] detailing why that individual is qualified and how they will contribute.
-After the RFC has been discussed, a unanimous vote by the PowerShell Committee will be required for the new Repository Maintainer to be confirmed.
+At any point in time, the existing Repository Maintainers can nominate a strong community member to become a Repository Maintainer.
+Nominations are brought to the PowerShell Committee to understand the reasons and justification.
```

Comment:
The updated wording of this makes the nomination process confusing. It implies that the Committee must approve a nomination without actually stating it. If the committee consultation is just advisory (does not require approval to add a new maintainer), then why is the committee consulted? Also, what constitutes approval? A majority or unanimous vote or maintainers or committee members? Should it be a unanimous vote of both maintainers and committee members? Majority of maintainers and unanimous committee? Unanimous maintainers and majority committee?